ocw/lib/ec2.py

```diff
@@ -173,7 +173,11 @@ def delete_vpc_subnets(self, vpc) -> None:
                     self.log_info(f'Deletion of {interface} skipped due to dry_run mode')
                 else:
                     self.log_info(f'Deleting {interface}')
-                    interface.delete()
+                    try:
+                        interface.delete()
+                    except ClientError as exc:
+                        self.log_err("delete_vpc_subnets: %s", exc)
```

Comment:
see real log above , this will lead to log entry : `2023-08-26 04:04:23,798 ocw.lib.ec2  ERROR    [sapha] delete_vpc_subnets: %s   `

to actually output content you need to : `self.log_err("delete_vpc_subnets: {}", exc)`